packages/modules/auth/src/index.ts

```diff
@@ -1,11 +1,11 @@
-import {
-  moduleDefinition,
-  revertMigration,
-  runMigrations,
-} from "./module-definition"
+import { moduleDefinition } from "./module-definition"
+import { initializeFactory, Modules } from "@medusajs/modules-sdk"
 
 export default moduleDefinition
-export { revertMigration, runMigrations }
 
-export * from "./initialize"
+export const initialize = initializeFactory({
```

Comment:
very nitpicky, but I think the `Factory` suffix leaks an implementation detail; I liked initialize better.

Reply:
If we can't use initialize, maybe we could go with `initializeModule`?

Reply:
This will disappear in the next commits. Basically we have everything to spin up module through medusaApp. So the module does not need to export the initialize and therefore the initialize will not even need to exists anymore 👍

Reply:
Also, i haven't changed the naming from what it was, initialize (for now) is still exported and the factory is not leaking outside of the module itself. Can you clarify on it please ☺️

Reply:
> This will disappear in the next commits. Basically we have everything to spin up module through medusaApp

Got it, even better!